Package.resolved

```diff
@@ -64,6 +73,15 @@
           "version": "0.4.1"
         }
       },
+      {
+        "package": "Kanna",
+        "repositoryURL": "https://github.com/tid-kijyun/Kanna.git",
+        "state": {
+          "branch": null,
+          "revision": "609367a2cd84827a33383cf7923cb4fe8f69ee0a",
+          "version": "5.2.2"
+        }
+      },
```

Comment:
This should have been resolved as per [this issue](https://github.com/SwiftGen/SwiftGen/issues/580).

I am not sure how many people use the old style, I can investigate it further. But from what I have gathered it should work when you run `brew install libxml2`

Reply:
Do you feel like this is worth fixing, @kwridan? I just worry I may need to go down the rabbit hole to resolve this 🐰 Otherwise, everything else should be resolved.

Reply:
No worries, not sure what can be done about it on this side :/ - it’s a transitive dependency issue

The reason I still use that command from time to time is due Xcode struggling to resolve dependencies when behind a company firewall.